extras/parse_travis_log.py

```diff
@@ -886,11 +886,12 @@ def build_return_code(status_vera_init,
     #       It may cause false positives. Even though cppcheck is executed
     #       and all its messages can be found in the log, cppcheck messages
     #       will not cause the build to fail.
-    #       To activate cppcheck, add following line to the if statement below.
-    #       Remove also the two strings '(DEACTIVATED)' behind cppcheck in
-    #       the summary_table above.
+    #       To activate cppcheck, add following lines to the if statement
+    #       below. Remove also the two strings '(DEACTIVATED)' behind
+    #       cppcheck in the summary_table above.
     #
     #       (status_cppcheck_init is None or status_cppcheck_init) and \
+    #       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
@gtrensch I see your point, and you are probably right that change will only happen one way here, cppcheck being included when evaluating whether a build was successful at some point. I still find this not very tidy. The main issue is how well the code will be understood by someone who was not involved in writing it, when code needs to be maintained later. In spite of our best intentions, cppcheck may still be deactivated five years from now ...